### PR TITLE
Java: BeanShell Injection

### PR DESCRIPTION
**Pull request by haby0**
BeanShell is a small, free, embeddable Java source interpreter with object scripting language 
features, written in Java. BeanShell dynamically executes standard Java syntax and extends it 
with common scripting conveniences such as loose types, commands, and method closures like 
those in Perl and JavaScript.

At present, BeanShell is also used in the Spring framework.

### CODE REVIEW

**Review by smowton — APPROVED** [review]

File: java/ql/src/experimental/Security/CWE/CWE-094/BeanShellInjection.qhelp [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+If it is not possible, BeanShell expressions should be run in a sandbox that allows accessing only
+explicitly allowed classes.
+</p>
+</recommendation>
+
+<example>
+<p>
+The following example uses untrusted data to build and run a BeanShell expression.
+</p>
+<sample src="BeanShellInjection.java" />
```
Comment (smowton) [inline java/ql/src/experimental/Security/CWE/CWE-094/BeanShellInjection.qhelp:26]:
```suggestion
<sample src="BeanShellInjection.java" />
</example>
```

**Review by smowton — COMMENTED** [review]
This is also missing an `options` file for the test to bring the `bsh` stubs into scope

**Comment by haby0** [comment]
@smowton I made changes, please review again, thank you!

**Review by smowton — APPROVED** [review]

**Comment by haby0** [comment]
@smowton Thank you!

**Pull request merged**